src/pages/dataset-view/dataset-view.js

```diff
@@ -43,6 +45,10 @@ export default class extends Component {
 
         const systemName = this.props.dataset.systemName
 
+        if (this.props.dataset.sourceType == "remote") {
+            return (<DatasetDetailView match={this.props.match} />)
+        }
```

Comment:
(Tony) this might be a good candidate for a selector rather than looking at the `sourceType` string down in the presenter component.  Maybe a selector called `isQueryable`?  If it's false then only show the detail view.